### PR TITLE
[@selrond/knit] Added definitions

### PR DESCRIPTION
**Pull request by ChocolateLoverRaj**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## selrond__knit ([<kbd>unpkg</kbd>](https://unpkg.com/browse/selrond__knit@latest/))
was missing the following properties:
1. knit
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 74e4f2a945cb8b0d6f6241820e381014148450e4
</p>

File: types/selrond__knit/selrond__knit-tests.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import knit = require('selrond__knit');
```
Comment (peterblazejewicz) [inline types/selrond__knit/selrond__knit-tests.ts:1]:
```suggestion
import knit = require('@selrond/knit');
```

File: types/selrond__knit/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (peterblazejewicz) [inline types/selrond__knit/tsconfig.json:1]: you have to update this file to support scoped package paths, as per readme:
https://github.com/DefinitelyTyped/DefinitelyTyped#what-about-scoped-packages

File: types/selrond__knit/selrond__knit-tests.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+// $ExpectType number[]
+knit.knit([10, 9, 8], 3);
```
Comment (peterblazejewicz) [inline types/selrond__knit/selrond__knit-tests.ts:4]:
```suggestion
knit([10, 9, 8], 3);
```

File: types/selrond__knit/selrond__knit-tests.ts [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+// $ExpectType (string | number)[]
+knit.knit([1, 2, 3], ' + ');
```
Comment (peterblazejewicz) [inline types/selrond__knit/selrond__knit-tests.ts:7]:
```suggestion
knit([1, 2, 3], ' + ');
```

File: types/selrond__knit/selrond__knit-tests.ts [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+
+// @ts-expect-error
+knit.knit([1, 2, 3]);
```
Comment (peterblazejewicz) [inline types/selrond__knit/selrond__knit-tests.ts:10]:
```suggestion
knit([1, 2, 3]);
```

File: types/selrond__knit/selrond__knit-tests.ts (hunk 3 above)
Comment (ChocolateLoverRaj) [reply types/selrond__knit/selrond__knit-tests.ts:4]: But knit is a named export

File: types/selrond__knit/selrond__knit-tests.ts [hunk 6]
```diff
@@ -0,0 +1,10 @@
+import knit = require('@selrond/knit');
```
Comment (peterblazejewicz) [inline types/selrond__knit/selrond__knit-tests.ts:1]:
```suggestion
import { knit } from '@selrond/knit';
```
than other changes would work (using named import directly)

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@ChocolateLoverRaj thanks!

**Comment by ChocolateLoverRaj** [comment]
Ready to merge

**Pull request merged**